### ISSUE TITLE
Syntax coloring

### ISSUE THREAD

**Issue by johs** [ermouth/cloudwall#22]
JS syntax coloring not aware of templates in my favorite app Ddoc lab
<img width="311" alt="screen shot 2018-04-26 at 14 32 46" src="https://user-images.githubusercontent.com/48370/39310675-53e7d014-496b-11e8-962e-898af284d37b.png">

**Comment by ermouth** [comment]
Yes, it‘s known problem of CodeMirror parser, which was fixed very recently. So fix is not yet propagated to public versions of CloudWall and Ddoc Lab. I‘ll get you know when update is ready.

If template literals highlighting is critical right now, consider doing these steps:
* get https://github.com/codemirror/CodeMirror/blob/master/mode/javascript/javascript.js
* open /lib/plugins.js CloudWall ddoc attachment
* find string `//codemirror javascript` in plugins.js
* replace code block following `//codemirror javascript` comment with updated highlighter you got from github
* important: be careful not to remove subsequent parsers in plugins.js accidentally
* save attachment back and reload CloudWall.

**Comment by johs** [comment]
Not critical:)
Easy to isloate in the js code with templates as ddoc fragments too
`(d,i) => {/*{{.template.hello}}*/}`

Another ES6 detail: Uglify in JS resources seems to ignore `let` and `const`
And how about an ES6 transpiler as a feature on JS resources?:)

**Comment by ermouth** [comment]
> Uglify in JS resources seems to ignore let and const

Can you please give an example, how it looks like? Btw, do you use CloudWall or Ddoc lab standalone? And of which version?

>And how about an ES6 transpiler as a feature on JS resources?

Very unlikely. I‘ve made some experiments a year ago, results where disappointing: enormously bloated code, running pretty slow.

**Comment by johs** [comment]
Thanks,
I use Ddoc lab 1.9.20 in CloudWall 2.2.3, couchapp build
this is how a build looks like without and with uglifyer
 
<img width="624" alt="screen shot 2018-04-27 at 07 37 37" src="https://user-images.githubusercontent.com/48370/39346706-c99f6766-49ef-11e8-960c-79189f54e87c.png">
<img width="863" alt="screen shot 2018-04-27 at 07 38 01" src="https://user-images.githubusercontent.com/48370/39346740-f0da0d9a-49ef-11e8-99f3-2a8316ee7b13.png">
Looks like `const` screws things up.

**Comment by ermouth** [comment]
> Looks like `const` screws things up.

It doesn‘t. Your const is removed since it‘s dead code, never used. However, if you use `b` somewhere in expression, it will be inlined in expression by value, since its value is precisely known during uglify )

**Comment by johs** [comment]
Thanks! Uglify really smart too:)
